src/filesystem.hpp

```diff
@@ -146,6 +146,44 @@ class FileSystem {
 
         *local_path = std::string(tmp_folder);
 
+        return StatusCode::OK;
+    }
+#elif _WIN32
+    static StatusCode createTempPath(std::string* local_path) {
+        if (!local_path)
+            return StatusCode::FILESYSTEM_ERROR;
```

Comment:
Lets add log here that target temp path is not set: SPDLOG_LOGGER_ERROR(modelmanager_logger, "Target path variable for createTempPAth not set. {}"

Reply:
following the commit and the modify the code